mods/d2k/rules/infantry.yaml

```diff
@@ -108,3 +107,124 @@ MEDIC:
 	AttractsWorms:
 		Intensity: 180
 
+fremen:
+	Inherits: ^Infantry
+	Valued:
+		Cost: 800
+	Tooltip:
+		Name: Fremen
+		Description: Elite sniper infantry unit\n  Strong vs Infantry\n  Weak vs Vehicles\n  Special Ability: Invisibility
+	Buildable:
+		Queue: Infantry
+		BuildPaletteOrder: 85
+		Prerequisites: ~barr.atreides, palace, ~techlevel.high
```

Comment:
We also do the same thing on bleed, so this is best left to a specific followup PR.